components/x-topic-search/__tests__/x-topic-search.test.jsx

```diff
@@ -43,6 +44,18 @@ describe('x-topic-search', () => {
 		});
 	});
 
+	describe('changed from on blur to on focus', () => {
+		it('should call select on event.target', () => {
+			const selectMock = sinon.spy();
```

Comment:
Can you use Jest for the spy, so we're not introducing another dependency?

https://jestjs.io/docs/en/jest-object